bin/yarn

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
+APP_ROOT = File.expand_path('..', __dir__)
+Dir.chdir(APP_ROOT) do
+  begin
+    exec "yarnpkg", *ARGV
+  rescue Errno::ENOENT
+    $stderr.puts "Yarn executable was not detected in the system."
+    $stderr.puts "Download Yarn at https://yarnpkg.com/en/docs/install"
+    exit 1
```

Comment:
I thought of adding `npm install yarn` here, but it turns out it's impossible to install `yarn` without installing the rest of dependencies via `npm`. This is a default message, I think this behavior is good enough.